app/views/accounts/_validated_badge.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if @presenter.show_verified_badge? %>
+  <%= render BadgeTooltipComponent.new(icon: :check_circle, icon_class: 'text-success', tooltip_text: @presenter.biometric_identity_verification? ? t('components.badge_tooltip_component.verified_biometric') : t('components.badge_tooltip_component.verified'), class: 'usa-tooltip', data_position: 'top').with_content(t('headings.account.verified_account')) %>
```

Comment:
What does `data_position` do? It's not a valid `data-` attribute. I'm assuming we'd only want `data-position` attribute to be assigned.

Reply:
> What does `data_position` do? It's not a valid `data-` attribute. I'm assuming we'd only want `data-position` attribute to be assigned.

Removed in 53e440976f.